lib/bitters/generator.rb

```diff
@@ -1,37 +1,43 @@
-require 'bitters/version'
-require 'fileutils'
-require 'pathname'
-require 'thor'
+require "bitters/version"
+require "fileutils"
+require "pathname"
+require "thor"
 
 module Bitters
   class Generator < Thor
-    map ['-v', '--version'] => :version
-    map ['scaffold'] => :install
-    map ['delete'] => :remove
+    map ["-v", "--version"] => :version
+    map ["scaffold"] => :install
+    map ["delete"] => :remove
 
-    desc 'install', 'Install Bitters into your project'
-    method_options path: :string
+    desc "install", "Install Bitters into your project"
+    method_options path: :string, syntax: :string
     def install
       if bitters_files_already_exist?
         puts "Bitters files already installed, doing nothing."
       else
         install_files
-        puts "Bitters files installed to #{install_path}/base"
+        puts "Bitters files installed to #{install_path}/base."
+        if options[:syntax].to_s == "sass"
+          puts "Converted files to Sass syntax."
+        end
       end
     end
 
-    desc 'reset', 'Reset Bitters'
+    desc "reset", "Reset Bitters"
     def update
       if bitters_files_already_exist?
         remove_bitters_directory
         install_files
         puts "Bitters files updated."
+        if options[:syntax].to_s == "sass"
+          puts "Converted files to Sass syntax."
```

Comment:
This should be moved down to line 106 (after `convert_syntax_to_sass` is successful). When I had the check for `sass-convert` failing, it told me it failed but then it told me they were converted here.